.github/workflows/pre-commit.yml

```
@@ -27,13 +36,17 @@ jobs:
           key: ${{ runner.os }}-pip-${{ hashFiles('**/setup.py') }}
           restore-keys: |
             ${{ runner.os }}-pip-
+
       - name: Install dependencies
         run: |
               python -m pip install --upgrade pip
-              if [ -f requirements.txt ]; then pip install -r requirements.txt; fi
+              if [ -f requirements_dev.txt ]; then pip install -r requirements_dev.txt; fi
```

```suggestion
              if [[ -f requirements_dev.txt ]]; then pip install -r requirements_dev.txt; fi
```
See [here](https://github.com/ivadomed/ivadomed/wiki/variables#if-statements).

---

COOL. TIL. Thanks for the info!